sources/niCURL/src/CURL.cpp

```diff
@@ -1401,6 +1408,98 @@ class cCURL : public cIUnknownImpl<iCURL>
                                       iFetchSink* apSink,
                                       const tStringCVec* apHeaders)
   {
+    if (_hasFetchOverride) {
```

Comment:
This should move to its own function for clarity. Probably better to have _FetchWithOverride and _FetchEmscripten.